### DISCUSSION TITLE
Setting preset through cmd/powershell should not show the app

### DISCUSSION THREAD

**Discussion by ThunderSmotch** [maximmax42/Discord-CustomRP#145] [Ideas]
Can we have the launch option to have the app launch to tray/load the preset without the window popping up?
Or an option to launch minimized.

**Comment by ghost** [comment]
You can make it launch to tray by going to Settings -> Start Minimized
However, you can't load presets on a minimized window.

**Reply by ThunderSmotch** [reply]
That's exactly what I am trying to ask for.

Loading a preset while the window is minimized to tray. It should be possible.

**Comment by maximmax42** [comment]
Are you trying to import a preset through console when the app is already running or not?

**Reply by ThunderSmotch** [reply]
Yes, the app is already running on the tray.
I import a preset through the console and it loads the preset and shows the window.

Because my script is already loading the preset through a console command I do not need to see the window, it can be done in the background.

**Reply by maximmax42** [reply]
Hm. I guess I can add a flag for silent import.

**Reply by ThunderSmotch** [reply]
Yeah, something like --silent could work!

Also, thanks for the app, it's really useful and customizable :)
